shared/src/packages/surveys/libs/validation-schemas/survey-request.validation-schema.ts

```diff
@@ -0,0 +1,15 @@
+import joi from 'joi';
+
+import { SurveyValidationMessage } from '../enums/enums.js';
+import { type SurveyRequestDto } from '../types/types.js';
+
+const SurveyValidationSchema = joi.object<SurveyRequestDto, true>({
+  userId: joi.number().integer().required().messages({
+    'any.required': SurveyValidationMessage.USER_ID_REQUIRED,
+  }),
+  preferences: joi.array().items(joi.string()).required().messages({
```

Comment:
We also should validate that strings max length is 1000 and we have at least one item in array. Also move 1000 and 1 from the example below to SurveyValidationRule
```suggestion
  preferences: joi.array().items(joi.string().max(1000)).min(1).required().messages({
   'string.max': ...,
   'array.min': ...,
```